File: result/translate/transcoder/java2cpp/c++/DYNAMIC_PROGRAMMING_SET_8_MATRIX_CHAIN_MULTIPLICATION.cpp
int f_gold ( int p [ ] , int i , int j ) {
  if ( i == j ) return 0 ;
  int min = INT_MAX ;
  for ( int k = i ;
  k < j ;
  k ++ ) {
    int count = f_gold ( p , i , k ) + f_gold ( p , k + 1 , j ) + p [ i - 1 ] * p [ k ] * p [ j ] ;
    if ( count < min ) min = count ;
  }
  return min ;
}

File: dataset/python2java/python/DYNAMIC_PROGRAMMING_SET_8_MATRIX_CHAIN_MULTIPLICATION.py
def f_gold ( p , i , j ) :
    if i == j :
        return 0
    _min = sys.maxsize
    for k in range ( i , j ) :
        count = ( f_gold ( p , i , k ) + f_gold ( p , k + 1 , j ) + p [ i - 1 ] * p [ k ] * p [ j ] )
        if count < _min :
            _min = count
    return _min